src/strings-in-block/class-html.php

```diff
@@ -89,7 +91,9 @@ public function update( \WP_Block_Parser_Block $block, array $string_translation
 				isset( $string_translations[ $string_id ][ $lang ] ) &&
 				ICL_TM_COMPLETE == $string_translations[ $string_id ][ $lang ]['status']
 			) {
-				$block->innerHTML = $string_translations[ $string_id ][ $lang ]['value'];
+				$translation = $string_translations[ $string_id ][ $lang ]['value'];
+				$block = apply_filters( 'wpml_updating_block_translation', $block, $translation );
+				$block->innerHTML = $translation;
```

Comment:
@brucepearson I think you can extract the two lines into a private method.

Something like that:

```php
private function filter_block_translation( $block, $translation ) {
	$block            = apply_filters( 'wpml_updating_block_translation', $block, $translation );
	$block->innerHTML = $translation;

	return $block;
}
```

Then here an above you can call `$block = $this->filter_block_translation( $block, $translation );`.

(or even pass `$block` by reference and avoid to return it)